src/interaction/ConstrainCOM.py

```diff
@@ -0,0 +1,107 @@
+#  Copyright (C) 2016
+#      Max Planck Institute for Polymer Research
+#  
+#  This file is part of ESPResSo++.
+#  
+#  ESPResSo++ is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  
+#  ESPResSo++ is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+
+
+r"""
+***********************************************
+**espressopp.interaction.ConstrainCOM**
+***********************************************
+This class is for calculating forces of constraining center of mass of subchains.
+Subchains are defined as a tuple list.
```

Comment:
there should be somewhere here a reference to the article (as it is in ConstrainCOM.hpp:40)